.github/workflows/build.yml

```diff
@@ -0,0 +1,326 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
```

Comment:
We want the pipeline to only run on demand.  Take these steps to enable the `repository_dispatch` GitHub webhook [1].

* Modify the `on` YAML dictionary so it has an empty entry named
  `repository_dispatch`.
  
* Create a GitHub personal access token [2].

* Ensure GitHub actions is enabled for the repo.  It is enabled by default, but someone may have disabled it somewhere along the line. So make sure it is enabled. [3]
  
* Once the `build.yml` and all the files on which it depends have been committed to the repo, then the following CURl command will cause the pipeline to run.
  
  ```bash
  PERSONAL_ACCESS_TOKEN=<GITHUB_PERSONAL_ACCESS_TOKEN>
  REPO_NAME=WASdev/azure.websphere-traditional.image
  curl --verbose -X POST https://api.github.com/repos/${REPO_NAME}/dispatches -H 'Accept: application/vnd.github.everest-preview+json' -H 'Authorization: token ${PERSONAL_ACCESS_TOKEN} --data '{"event_type": "run"}'
  ```

[1] https://mainawycliffe.dev/blog/github-actions-trigger-via-webhooks

[2] https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token

[3] https://docs.github.com/en/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository

Reply:
Done. And the usage is added as comments in the workflow